presto-cli/pom.xml

```diff
@@ -30,6 +30,12 @@
         <dependency>
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-client</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.facebook.airlift</groupId>
+                    <artifactId>json</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
why add this exclusion?

Reply:
There is a dependency scope error as follows...

```
[ERROR] Found a problem with test-scoped dependency com.facebook.airlift:json
[ERROR] Scope compile was expected by artifact com.facebook.presto:presto-client:0.291-SNAPSHOT
[ERROR]
[ERROR] Dependency chain:
[ERROR] com.facebook.presto:presto-cli:jar:0.291-SNAPSHOT
[ERROR] \- com.facebook.presto:presto-client:jar:0.291-SNAPSHOT:compile
[ERROR]    \- com.facebook.airlift:json:jar:0.215:compile
```

It is including json as a transitive dependency in the compile scope when it is only needed in the test scope

Reply:
Since the the json dependency is likely required by the `presto-client` jar we don't want to exclude it. For this case and any others that you come across in the future, the solution is to just add it to the set of ignored dependencies like in this commit: https://github.com/prestodb/presto/pull/23979/commits/024e1dbeba54cccd8d97526238d8f39b8aa05f38